libs/wire-api/test/unit/Test/Wire/API/Roundtrip/Aeson.hs

```diff
@@ -206,6 +206,7 @@ tests =
       testRoundTrip @(Team.Feature.TeamFeatureStatus 'Team.Feature.TeamFeatureConferenceCalling),
       testRoundTrip @(Team.Feature.TeamFeatureStatus 'Team.Feature.TeamFeatureSelfDeletingMessages),
       testRoundTrip @Team.Feature.TeamFeatureStatusValue,
+      testRoundTrip @Team.Feature.PaymentStatus,
```

Comment:
I'm assuming you deliberately skipped `PaymentStatusValue` since it's implicit?

Reply:
Ye, but I will add it, too. Doesn't hurt.